### DISCUSSION TITLE
Status Dec 28, 2020

### DISCUSSION THREAD

**Discussion by robojay** [dmitryelj/SoapyICR8600#6] [General]
I've been chipping away at the driver over the past week or so.

I'm building with SoapySDR from source, but from a commit back in January 2020 (https://github.com/pothosware/SoapySDR/commit/7ad496133a09ec803cb2dcae936dcca9f6799875) that seems to align best with the January 2020 release of the PothosSDR package (https://downloads.myriadrf.org/builds/PothosSDR/)

Using the PothosSDR package eliminates the need to build a handful of tools from source, as long as the SoapySDR versions (and device DLLs) are in sync.

I have been able to use soapySDRutil, gqrx, and quisk with the driver.  I've also been able to use the driver from Python.

At least in my setup, the IC-R8600 somewhat works - although I'm fairly certain there is some data dropout occurring (I hear pops and somewhat distorted audio occasionally).  Quisk barely works - their SoapySDR support is still in early stages and at this point I'd rather work with gqrx as a baseline. 

----
**IMPORTANT**:  PothosSDR **JUST RELEASED** a new package **TODAY**.  I will need to rebuild and test things with the new package...

**Comment by robojay** [comment]
1. Updated my SoapySDR to the current master.
2. Rebuilt SoapySDR
3. Rebuilt SoapyICR8600
4. Installed the latest PothosSDR (https://downloads.myriadrf.org/builds/PothosSDR/PothosSDR-2020.12.28-vc14-x64.exe)
5. Copied the icr8600Support.dll to ..Program Files\PothosSDR\lib\SoapySDR\modules0.8
6. Ran gqrx and it works (still with issues noted above)

Looks like things are in sync on December 28, 2020!

**Reply by dmitryelj** [reply]
Cool, thanks for sharing. Have a nice New Year :)

**Comment by robojay** [comment]
Assumptions... Windows 10, Visual Studio 2019, Icom IC-R8600 I/Q drivers installed
